src/foam/dao/MDAO.java

```diff
@@ -107,11 +107,44 @@ public MDAO(ClassInfo of) {
     index_ = new AltIndex(new TreeIndex((PropertyInfo) this.of_.getAxiomByName("id")));
   }
 
+
+
+  public ArrayList<PropertyInfo> propertyInfoList_(Index index){
```

Comment:
These look like an internal method, I think it should be private or protected.  Also @kgrgreer what's the style policy for methods with a `_` suffix?

Reply:
We use the Doug Lea's coding standard (http://gee.cs.oswego.edu/dl/html/javaCodingStd.html) and name protected/private variables with a trailing _. Even on the JS side, where you don't have package/private/protected, we mostly use a trailing _ to denote that it should be considered an internal method or property. Even more important in JS, since you don't have the language support to denote internal features. But you shouldn't have anything be public and have an trailing _.